core/src/main/java/org/infinispan/container/entries/ReadCommittedEntry.java

```diff
@@ -321,4 +327,31 @@ public boolean undelete(boolean doUndelete) {
       return false;
    }
 
+   public static class Externalizer extends AbstractExternalizer<ReadCommittedEntry> {
```

Comment:
Hahah, that'd be a little nasty surprise to find at runtime. I don't think there's a "need" per-se to serialize them, but it can be convenient for the user. See `FunctionalMapTest.replaceWithVersion` method, where I return the entry view and I do some assertions on the value and the metadata. If the entry view was not serializable and wanted to return value and metadata, I'd have to come up with my own data holder to send both things back, or send them in a list or array (urrghhh)